labelbox/data/annotation_types/classification/classification.py

```diff
@@ -35,6 +36,7 @@ class ClassificationAnswer(FeatureSchema, ConfidenceMixin):
     """
     extra: Dict[str, Any] = {}
     keyframe: Optional[bool] = None
+    classifications: List['ClassificationAnnotation'] = []
```

Comment:
note this means `classifications=[]` will be present in our json. But easier on coding. LMK if it is an issue